common/djangoapps/entitlements/api/v1/tests/test_views.py

```diff
@@ -117,7 +117,7 @@ def test_non_staff_get_select_entitlements(self):
         )
         assert response.status_code == 200
 
-        results = response.data.get('results', [])
+        results = response.data.get('results', [])  # pylint: disable=E1101
```

Comment:
Note: This was added for a Quality pylint error that did not make any sense.  I have attempted to clean it up, but there seems to be no solution.  I can explain more in person.